packages/react/src/lib/commonPropInterfaces.ts

```diff
@@ -18,6 +18,8 @@ export interface AnimatedComponentProps {
 export interface UIComponentProps<P = any, V = any>
   extends StyledComponentProps<P, V>,
     AnimatedComponentProps {
+  [key: string]: any
```

Comment:
All of our components (except, `Flex`) that actually should have extendable props use this interface